src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -614,6 +634,28 @@ public void setImplVelocity(Vector3d velocity) {
     }
 
     @Override
+    public Map<Statistic, Long> getStatistics() {
+        return ((IMixinStatisticHolder) statsFile).getStatistics();
+    }
+
+    @Override
+    public void setStatistics(Map<Statistic, Long> statistics) {
+        ((IMixinStatisticHolder) statsFile).setStatistics(statistics);
+        // statsFile.saveStatFile();
```

Comment:
I was not sure whether the changes should be saved instantly or whether I should wait for it to be saved automatically.